src/main/java/edu/cornell/kfs/concur/batch/ConcurSaeDelimitedFlatFileSpecification.java

```diff
@@ -35,22 +34,29 @@ public void parseLineIntoObject(
         int index = -1;
         for (final String lineSegment : lineSegments) {
             index++;
-            if (StringUtils.isEmpty(lineSegment)) {
-                cleanedSegments[index] = lineSegment;
-                continue;
-            }
-            String cleanedSegment = TABS_PATTERN.matcher(lineSegment).replaceAll(KFSConstants.BLANK_SPACE);
-            final Matcher removableCharsMatcher = REMOVABLE_CHARS_PATTERN.matcher(cleanedSegment);
-            if (removableCharsMatcher.find()) {
-                LOG.warn("parseLineIntoObject, Found special characters or extra quotes on SAE line {} at column {}",
-                        lineNumber, index + 1);
-                saeLine.addColumnNumberContainingSpecialCharacters(index + 1);
-                removableCharsMatcher.reset();
-                cleanedSegment = removableCharsMatcher.replaceAll(KFSConstants.EMPTY_STRING);
+            String cleanedSegment;
```

Comment:
The updated code looks good but I think most of the code inside the for look could go into a separate method called something along the lines 'cleanSegment' or 'cleanText'? Thank you

Reply:
I've moved more of the logic into another method as requested.